packages/dispatcher-converter/src/converter/singleFilesConverter.js

```diff
@@ -418,10 +448,28 @@ class SingleFilesConverter {
         );
     }
 
+    getAllVhosts() {
+        let allFiles = [];
+        let vhostFiles = this.config.vhostsToConvert;
+
+        vhostFiles.forEach((file) => {
+            if (fs.lstatSync(file).isFile()) {
+                allFiles.push(file);
+            } else if (fs.lstatSync(file).isDirectory()) {
+                let globPattern = file + "/**/*";
```

Comment:
You should ALWAYS use `path.join()` for ALL path concatenations, this will adapt the paths as per the underlying OS path separator. The above might not work in Windows env.

Reply:
Done